### DISCUSSION TITLE
Fontsource font deployement failed

### DISCUSSION THREAD

**Discussion by loudpose** [vercel/community#2062] [Bug Report]
Hi,

I couldn't find Fontsource community at https://vercel.com/help#issues, so reporting this issue here. I use the Gloock font (Google Fonts) for my brand, but I discovered that it's not available at npm i @fontsource/gloock

After a minor investigation, I noticed that the deployment for this particular font has failed. Here's the link https://github.com/fontsource/fontsource/tree/main/fonts/google/gloock

Could you please look into it?

Fontsource is an amazing tool!

Thank you,

**Comment by amyegan** [comment]
Hi @loudpose! It looks like there is already an open issue in the Fontsource repo related to Gloock (https://github.com/fontsource/fontsource/issues/657) and there may be a workaround (https://github.com/fontsource/fontsource/issues/636#issuecomment-1474879365). Please follow up there for the latest info
